### PR TITLE
Add Governance section

### PR DESCRIPTION
**Pull request by jarrodmillman**
@seberg could you review this?

### CODE REVIEW

**Comment by jarrodmillman** [comment]
@seberg I am about to do a big revision.  So please hold off reviewing for a bit.

**Comment by jarrodmillman** [comment]
@seberg done.  Let me know if you have any suggestions or corrections.  We are trying to keep the article to at most 3,500 words.  So this is about the right length for this section.  We will add something to the next section (i.e., discussion) about our interest in growing (and plans to grow) the community.

File: paper.tex [hunk 1]
```diff
@@ -592,14 +584,31 @@ \subsection*{Maintainers and contributors}
...
-
-% may be useful:
-% - https://github.com/docathon/watchtower
-% - https://github.com/machine-shop/oss_devkit/blob/master/plot_pr.py % (PR over time plot, there may be other better versions out there)
+% https://mail.python.org/pipermail/numpy-discussion/2015-October/073849.html
+% https://github.com/numpy/numpy/pull/6352
+NumPy adopted an official Governance Document on October~5,
+2015\cite{NumPyProjectGovernance}.
+All project decisions are made by consensus of interested contributors.
+This means everyone is entrusted with veto power.
```
Comment (stefanv) [inline paper.tex]: Is this true?

Comment (jarrodmillman) [reply paper.tex]: In the "Consensus-based decision making by the community" section of the
[NumPy project governance and decision-making](https://numpy.org/devdocs/dev/governance/governance.html), it states:

> Normally, all project decisions will be made by consensus of all interested Contributors. 

and 

> For us, what consensus means is that we entrust everyone with the right to veto any change if they feel it necessary.

Comment (jarrodmillman) [reply paper.tex]: How about:

> Project decisions are usually made by consensus of interested contributors.  This means for many thing everyone is entrusted with veto power.

Comment (seberg) [reply paper.tex]: "Usually" or "Normally" is good here I think. The docs also say: "the [steering] council may, if necessary, make decisions when regular community discussion doesn’t produce consensus on an issue in a reasonable time frame." Which I think practically can take the community out of the equation.

We could possible even say "interested parties" or anyone in the community (although contributors can be interpreted slightly more restrictive). I am thinking if the second sentence would also make sense as saying the steering council may make the decisions if necessary.

**Pull request merged**